parent/pom.xml

```diff
@@ -510,7 +507,7 @@
     </distributionManagement>
 
     <properties>
-        <java.version>11</java.version>
+        <java.version>17</java.version>
```

Comment:
Will this make 4.1 JDK 17 only for users?

Reply:
It would.

Reply:
Do you need Quarkus on Java 11?

Reply:
Nope i was just making sure 4.1 minimum version would be 17 now.  I know 4.0 we are keeping at 11 right?

Reply:
Correct -- 4.0 is still 11.   4.1 is now part of EE11 which requires a minimum of Java 17.